bin/cyhy-tool

```diff
@@ -262,9 +262,9 @@ def main():
 
         for orgId in orgIds:
             retire(db, orgId)
-        print "Do not forget to move " + str(
+        print str(
             orgIds
-        ) + ' to the "Retired CyHy Orgs" tab in the Google Spreadsheet!'
+        ) + ' has been successfully retired.'
```

Comment:
Single quotes are no longer needed now that there are no double quotes being printed out.
```suggestion
        ) + " has been successfully retired."
```